app/views/spree/admin/orders/_invoice_table2.html.haml

```diff
@@ -47,12 +47,12 @@
         %strong= @order.has_taxes_included ? t(:total_incl_tax) : t(:total_excl_tax)
       %td{:align => "right", :colspan => "2"}
         %strong= @order.has_taxes_included ? @order.display_total : display_checkout_total_less_tax(@order)
-    - display_checkout_taxes_hash(@order).each do |tax_rate, tax_value|
+    - display_checkout_taxes_hash(@order).each do |tax|
       %tr
         %td{:align => "right", :colspan => "3"}
-          = t(:tax_total, rate: tax_rate)
+          = t(:tax_total, rate: tax[:percentage])
```

Comment:
It could be interesting to display these lines in a ordered fashion, like according to tax rate, or so. Currently, we have this (see pic below), but I think it could fall out of scope for this issue, I can open an issue for this.

![image](https://user-images.githubusercontent.com/49817236/187735340-f21b5b2e-13e1-4f14-bf65-07f24948f710.png)

Reply:
Good idea. I added a commit for that.